instrumentation/grpc-1.5/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/grpc/v1_5/GrpcClientBuilderBuildInstrumentation.java

```diff
@@ -46,16 +45,7 @@
     @Advice.OnMethodEnter(suppress = Throwable.class)
     public static void addInterceptor(
         @Advice.FieldValue("interceptors") List<ClientInterceptor> interceptors) {
-      boolean shouldRegister = true;
-      for (ClientInterceptor interceptor : interceptors) {
-        if (interceptor instanceof TracingClientInterceptor) {
-          shouldRegister = false;
-          break;
-        }
-      }
-      if (shouldRegister) {
-        interceptors.add(0, TracingClientInterceptor.newInterceptor());
-      }
+      interceptors.add(0, GrpcInterceptors.CLIENT_INTERCEPTOR);
```

Comment:
We aren't focusing on duplicate instrumentation yet so I just removed what probably wasn't working anyways due to shading.